### DISCUSSION TITLE
Yarn.Lock Files

### DISCUSSION THREAD

**Discussion by Tajae1532** [openrewrite/rewrite#4737] [Q&A] [answered]
Hello, I've been looking into how to identify yarn.lock files. I have tried to process it as a sourceFile, PlainText, Quark, even looked across the documentation for help in regards to this but can not find anything. Any help would be nice.

**Comment by timtebeek** [comment, marked as answer]
hi! Unless you had explicitly passed in plain text masks, then those files are likely picked up as Quarks. I've added a change to our plugins to pick them up as plain text instead, such that you can target them with recipes.
https://github.com/openrewrite/rewrite-maven-plugin/commit/3a2a8031b9bb0db9ae2ac76f418b068ddab6a850
https://github.com/openrewrite/rewrite-gradle-plugin/commit/3218669cfa8ca11e3ddff4419aea11d165088d52

Use [our snapshot versions](https://docs.openrewrite.org/reference/snapshot-instructions) until the next release to get this change early. Hope that helps!
